src/selenium/speakerPage.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable no-empty-label */
+'use strict';
+
+const BasePage = require('./basePage.js');
+const By = require('selenium-webdriver').By;
+
+const SpeakerPage = Object.create(BasePage);
+
+SpeakerPage.searchTest = function() {
+  const idList = ['2330', '2240'];
+
+  return this.commonSearchTest('Mario', idList);
+};
+
+SpeakerPage.jumpToTrack = function() {
+  const self = this;
+  const speakerId = '2330';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 2.](https://app.codacy.com/manual/triplez/open-event-webapp/pullRequest?prid=3226522)